apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -200,22 +177,20 @@ public void Log_DefaultTraceId_TraceInformationIsNotLogged()
         // It is null by default but just to keep it easy to understand.
         var generator = Activity.TraceIdGenerator;
 
-        // Simulate the condition where the generated `TraceId` is equal to the default value of `ActivityTraceId`.
-        Activity.TraceIdGenerator = () => default;
-        using var activity = new Activity("my-activity").Start();
-
-        // Reset the generator, so that other tests aren't impacted.
-        Activity.TraceIdGenerator = generator;
-        
-        var logEntry = new LogEntry<string>(LogLevel.Information, "LogCategory", new EventId(1), "test", exception: null, (state, exception) => state);
-        var writer = new StringWriter { NewLine = "\n" };
-        
-        formatter.Write(logEntry, scopeProvider: null, writer);
-
-        var expectedJson = "{\"message\":\"test\",\"category\":\"LogCategory\",\"severity\":\"INFO\"}\n";
-        var actualJson = writer.ToString();
-
-        Assert.Equal(expectedJson, actualJson);
+        try
+        {
+            // Simulate the condition where the generated `TraceId` is equal to the default value of `ActivityTraceId`.
```

Comment:
I still don't like this. We need to check for the trace Id to be set or not in our production code, but given that it being unset is unlikely to happen (or there would be a bug in Activity) I think I'd prefer not to test this.

If we were to test it, I'd introduce an internal ActivityProvider of sorts that would default to Activity.Current, but that would allow for better testability.